src/modules/launcher/PowerLauncher/SettingsWatcher.cs

```diff
@@ -36,6 +37,8 @@ public SettingsWatcher(Settings settings)
             // Set up watcher
             _watcher = Microsoft.PowerToys.Settings.UI.Lib.Utilities.Helper.GetFileWatcher(PowerLauncherSettings.ModuleName, "settings.json", OverloadSettings);
 
+            _themeManager = new ThemeManager(App.Current);
```

Comment:
We should use the theme manager initialized during startup. This is to ensure that only a single state for theme is maintained across the program. You can modify `SettingsWatcher` constructor to pass `ThemeManager` initialized in App.xaml.cs.